tracex_project/extraction/urls.py

```diff
@@ -4,6 +4,7 @@
 from . import views
 
 urlpatterns = [
+    path('download-xes/', views.DownloadXesView.as_view(), name='download_xes'),
```

Comment:
you only need this if you are referencing on a new url. Since you only open a popup on the same url and not redirecting, you dont need this

Reply:
but please correct me if you actually need it and use it somewhere

Reply:
The idea is that you will be redirected if an error occurs: for example, if there is an error, you would receive an http response

for example:

HttpResponse("One or more files could not be found.", status=404)

or HttpResponse("No file type specified.", status=400)

I can change it if you want

Reply:
In general exception handling is a good thing. I think a better solution would be to create one Error Page in the task #95 where we display the error based on which error occurs. Else following the logic we need for every functionality an additional page. 
Could you please remove it and add a comment in #95 please?